Spanner/src/Admin/Database/V1/Client/DatabaseAdminClient.php

```diff
@@ -184,6 +184,9 @@ public function getOperationsClient()
     public function resumeOperation($operationName, $methodName = null)
     {
         $options = isset($this->descriptors[$methodName]['longRunning']) ? $this->descriptors[$methodName]['longRunning'] : [];
+        // OperationResponse is a V1 surface class, we need V2 surface class.
+        // [V2 LongRunning/Client/OperationsClient, getOperation]
+        // Other clients have also used the OperationResponse similarly.
         $operation = new OperationResponse($operationName, $this->getOperationsClient(), $options);
```

Comment:
These changes can be removed once the migration mode is fixed.

Reply:
We should modify the `MIGRATION_MODE` locally and generate the client as part of this PR, so we can preview those changes. Then we can merge the CL to change the migration mode once this PR is merged

Reply:
Discussed internally that Brent will take care of the `MIGRATION_MODE`, Thanks!